dashboard.py

```diff
@@ -1,4 +1,5 @@
 import pandas as pd
+import numpy as np
```

Comment:
I think this adds a new dependency to include in `requirements.txt`, right?

Reply:
pandas is built on top of numpy

Reply:
No, `numpy` is included in `pandas`.